src/test/graph_export_test.cpp

```diff
@@ -147,26 +254,73 @@ class GraphCreationTest : public TestWithTempDir {
 };
 
 TEST_F(GraphCreationTest, positiveDefault) {
-    ovms::GraphSettingsImpl graphSettings;
+    ovms::HFSettingsImpl hfSettings;
     std::string graphPath = ovms::FileSystem::appendSlash(this->directoryPath) + "graph.pbtxt";
     std::unique_ptr<ovms::GraphExport> graphExporter = std::make_unique<ovms::GraphExport>();
-    auto status = graphExporter->createGraphFile(this->directoryPath, graphSettings);
+    auto status = graphExporter->createServableConfig(this->directoryPath, hfSettings);
     ASSERT_EQ(status, ovms::StatusCode::OK);
 
     std::string graphContents = GetFileContents(graphPath);
     std::cout << graphContents << std::endl;
     ASSERT_EQ(expectedDefaultGraphContents, graphContents);
 }
 
+TEST_F(GraphCreationTest, rerankPositiveDefault) {
+    ovms::HFSettingsImpl hfSettings;
+    hfSettings.task = ovms::rerank;
+    hfSettings.rerankGraphSettings.targetDevice = "GPU";
+    hfSettings.rerankGraphSettings.modelName = "myModel";
+    hfSettings.rerankGraphSettings.numStreams = 2;
+    hfSettings.rerankGraphSettings.maxDocLength = 18;
+    hfSettings.rerankGraphSettings.version = 2;
+    std::string graphPath = ovms::FileSystem::appendSlash(this->directoryPath) + "graph.pbtxt";
+    std::string subconfigPath = ovms::FileSystem::appendSlash(this->directoryPath) + "subconfig.json";
+    std::unique_ptr<ovms::GraphExport> graphExporter = std::make_unique<ovms::GraphExport>();
+    auto status = graphExporter->createServableConfig(this->directoryPath, hfSettings);
+    ASSERT_EQ(status, ovms::StatusCode::OK);
+
+    std::string graphContents = GetFileContents(graphPath);
+    std::cout << graphContents << std::endl;
+    ASSERT_EQ(expectedRerankGraphContents, graphContents);
```

Comment:
In gtest you could always do sth like:
```suggestion
    ASSERT_EQ(expectedRerankGraphContents, graphContents) << graphContents;
```